man/neptune_log.Rd

```diff
@@ -0,0 +1,45 @@
+\name{neptune_log}
```

Comment:
Doc string and examples should be based on this:
https://github.com/neptune-ai/neptune-client/blob/c245bcaf7853e4366957b556dd5cb9487405103b/neptune/new/handler.py#L183